### ISSUE TITLE
Improve naming

### ISSUE THREAD

**Issue by franziskuskiefer** [AeneasVerif/eurydice#18]
I (Lucas) will spend half an hour on that this morning (Monday) to see if that's very quick to fix, otherwise I will work on that next week.

**Comment by franziskuskiefer** [comment]
@W95Psp what's the state here? Is this still in progress?

**Comment by W95Psp** [comment]
Yes, that's in progress. I spent some time on that on Monday, but it's not done yet, some WIP state can be found on branch https://github.com/AeneasVerif/eurydice/tree/lucas/improve-names.

**Comment by msprotz** [comment]
printing monomorphization information during extraction is done now, closing.
